examples/MQ/serialization/1-simple/testSerializationEx1.sh.in

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+source @CMAKE_BINARY_DIR@/config.sh
```

Comment:
```suggestion
source @CMAKE_BINARY_DIR@/config.sh -p
```

We need to prepend instead of overwriting to fix the failing linux tests.

Reply:
Fixed.